scripts/lib/sphinx/PythonObjectMeta.ts

```diff
@@ -10,15 +10,17 @@
 // copyright notice, and modified files need to carry a notice indicating
 // that they have been altered from the originals.
 
+export type pythonApiType =
```

Comment:
Types are capitalized. This requires changing more lines though

```suggestion
export type PythonApiType =
```